src/main/resources/assets/structurize/lang/en_us.json

```diff
@@ -106,5 +108,7 @@
     "structurize.config.windowcachecap.comment": "Sets the maximum number of parsed GUI window files to be stored for quick loading.",
     "structurize.gui.buildtool.creative_only": "Structurize does not support using the build tool when in survival. Switch to creative or install MineColonies and use the MineColonies Builder.",
     "structurize.gui.buildtool.unexpecteddatafixer": "Invalid datafixer detected! Side-effects possible! Check log for more info!",
-    "structurize.gui.replaceblock.ambiguous_properties": "Transformation from %s to %s is ambiguous because the following properties are not present in the source block: %s"
+    "structurize.gui.replaceblock.ambiguous_properties": "Transformation from %s to %s is ambiguous because the following properties are not present in the source block: %s",
+    "structurize.gui.buildtool.paste": "Instantly places the structure into the world, keeping Substitution Blocks.",
+    "structurize.gui.buildtool.pastenice": "Instantly places the structure into the world, as if constructed by a Builder."
```

Comment:
maybe specify a **MineColonies** builder?

Reply:
Well, theoretically anyone can make builder logic in their own mod depending on Structurize, so there is a possibility its not Minecolonies. Highly unlikely, but possible.